src/manual/peer2peer/js/main.js

```diff
@@ -1105,10 +1112,10 @@ function printHandler_(message, color) {
   }
   $('messages').innerHTML += '<span style="color:' + color + ';">' + message +
                             '</span><br>';
-  console.log(message);
   if (color === 'red') {
     throw new Error(message);
   }
+  console.log(message);
```

Comment:
nit: Why have you moved this line down? that way it will not be logged when actually throwing an error.